packages/common/src/Constants.js

```diff
@@ -31,5 +31,9 @@ module.exports = {
   ZERO_ADDRESS: "0x0000000000000000000000000000000000000000",
 
   // Names of different interfaces in the Finder.
-  interfaceName
+  interfaceName,
+
+  // Block number of first EMP created.
+  // https://etherscan.io/tx/0x741ccbf0f9655b0b71e3842d788d58770bd3eb80c8f5bdf4fdec7cd74a776ea3
+  UMA_FIRST_EMP_BLOCK: 10103723
```

Comment:
Nice use of this constants file, I was not aware of it, could be useful but it might require being more generalized to work for different networks, not sure, and I don't have any suggestions so I think this is fine for now.

Reply:
Yeah actually @daywiss that's a great point. What if instead of making `UMA_FIRST_EMP_BLOCK` an integer, you make it a mapping between `networkID` and integer, that way you can auto detect the first block based on the dev's specified network

Reply:
Annoyingly, it looks like Ganache uses the current time as the default `networkId` instead of some consistent number.

```
-i or --networkId: Specify the network id ganache-cli will use to identify itself (defaults to the current time or the network id of the forked blockchain if configured)
```
[https://github.com/trufflesuite/ganache-cli](https://github.com/trufflesuite/ganache-cli)

This should still work to differentiate between mainnet and everything else, though. Could have it use the mainnet block number if the networkId is 1 and use block number 0 otherwise. Will take a look at this.

Reply:
> Could have it use the mainnet block number if the networkId is 1 and use block number 0 otherwise. Will take a look at this.

I think this is very reasonable. Other public networks like Kovan are OK to start at 0

Reply:
I think changing the integer to a mapping could also remove the slightly annoying ask to set the FROM_BLOCK env var (which should still be an optional setting but not required anymore)

Reply:
Ooh, even better: https://web3js.readthedocs.io/en/v1.2.0/web3-eth-net.html#getnetworktype

This method knows how to recognize `ropsten` and `morden` and could probably be taught to recognize `kovan` through an upstream PR.